extension/src/background_scripts/background.js

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line no-undefined
```

Comment:
this is close, should be `no-undef`
anyways, no longer need (see my review comments)